modules/platforms/dotnet/Apache.Ignite.Core.Tests/Cache/CacheMetricsTest.cs

```diff
@@ -118,6 +120,16 @@ public void TestClusterGroupEnableStatistics()
             TestEnableStatistics(
                 cacheName,
                 (cache, b) => cache.Ignite.GetCluster().EnableStatistics(new[] {cacheName}, b));
+
+            var ignite = Ignition.GetIgnite();
```

Comment:
Can you please extract the code below into two separate tests? Something like:

* `TestClusterEnableStatisticsAllowsEmptyCacheNames`
* `TestClusterEnableStatisticsThrowsOnInvalidCacheName`